test/cmd.js

```diff
@@ -1,8 +1,11 @@
-const path = require('path')
-const test = require('tape')
-const crossSpawn = require('cross-spawn')
+import { dirname, join } from 'node:path'
+import { fileURLToPath } from 'node:url'
```

Comment:
Same comment as above, we could use the [URL API](https://nodejs.org/dist/latest-v16.x/docs/api/url.html).

Reply:
will fix things tomorrow